tests/unit/flow/test_flow_visualization.py

```diff
@@ -0,0 +1,33 @@
+import os
+import numpy as np
+from PIL import Image
+from jina.flow import Flow
+import glob
+
+
+def test_visualization():
+    cur_dir = os.path.dirname(os.path.abspath(__file__))
+
+    flow = (Flow().add(name='pod_a')
+            .add(name='pod_b', needs='gateway')
+            .join(needs=['pod_a', 'pod_b']))
+
+    flow.mermaidstr_to_jpg(path='flow_test1.jpg')
+    with Image.open(os.path.join(cur_dir, 'flow_original_test1.jpg')) as flow_original:
```

Comment:
did u add more files that are actually used in the tests?

Reply:
I added an extra yml file in the tests that is almost the same as the cross modal, is that what you mean?

Reply:
`flow_test1.jpg` and `flow_test2.jpg` do not have to be added,  and `flow_original.jpg` is not used

Reply:
I added them because when they were not there, it was not passing the CI tests

Reply:
And the `flow_original` is used to compare it with the new img created for the same flow